.github/workflows/build-nightly-release.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v4
       with:
-        token: ${{ secrets.AUTO_UPDATE_VERSION_RINGO_TOKEN }}
+        ssh-key: ${{ secrets.AUTO_UPDATE_VERSION_RINGO_KEY }}
```

Comment:
Should we call this RINGO_KEY?

Maybe something along `NIGHTLY_AUTO_VERSION_UPDATE_DEPLOY_KEY`?

I feel that'd quicker convey the intents behind the key

Reply:
Fair 😅

Reply:
```suggestion
        ssh-key: ${{ secrets.NIGHTLY_VERSION_UPDATE_DEPLOY_KEY }}
```